src/components/Heading/Heading2.js

```diff
@@ -4,7 +4,7 @@ import React from 'react';
 
 import Heading from './';
 
-// We define `<H2>` for developer convenience.
+// We define `<Heading2>` for developer convenience.
```

Comment:
Why are we using `<Heading2` here instead of `<Heading 2>`? I'd expect something more akin to the latter, since it's all the same component (which we refer to as `<Heading>`!)

Actually, I just read from the docs that this is an _alternative_ syntax, which I am probably fine with and I suspect makes things like Markdown processing easier?